cypress.json

```diff
@@ -1,4 +1,10 @@
 {
   "baseUrl": "http://localhost:5000",
-  "video": false
+  "video": false,
+  "fixturesFolder": "client/cypress/fixtures",
+  "integrationFolder": "client/cypress/integration",
+  "pluginsFile": "client/cypress/plugins/index.js",
+  "screenshotsFolder": "client/cypress/screenshots",
+  "videosFolder": "client/cypress/videos",
+  "supportFile": "client/cypress/support/index.js"
```

Comment:
Cypress assumes that it's in the root folder? Maybe there is a way to set its path/context once to avoid having to maintain all these folders/files mapping?

Reply:
Perhaps `fileServerFolder: "client/"`?

Reply:
> Cypress assumes that it's in the root folder? Maybe there is a way to set its path/context once to avoid having to maintain all these folders/files mapping?

Yes :confused:, I didn't like that I had to change all of those either.

The way I see that could probably set the path/context is changing `npm run cypress open` and `npm run cypress run` to run from another path. However, the current option seems better than that. With cypress knowing it's not in the root folder, you can run still run it using other commands: `cypress open` (in case you have it global) or `npx cypress open`.

LMK if you had something simpler in mind

> Perhaps fileServerFolder: "client/"?

I thought about this one too. I actually didn't get what's its purpose, but changing it doesn't affect what we want. Cypress keeps thinking it's in the root folder and, since there's no cypress folder in there, it creates it in an empty state.